AAUnlimited/MemMods/AAPlay/Events/HInjections.cpp

```diff
@@ -67,6 +74,34 @@ bool __stdcall TickRedirect(ExtClass::HInfo* hInfo) {
 			loc_activeFaceXX = active->m_xxFace;
 			loc_passiveFaceXX = passive->m_xxFace;
 		}
+		// Detecting cumshot pose stage
+		cumshotStart = (byte*)ExtVars::ApplyRule(cumshotStartOffset);
```

Comment:
Does this address change during the sex scene or is it static the whole time?

Reply:
Static for all time. This address recommend me k4. Thanks to him.